src/accessibility-and-localization/accessibility.md

```diff
@@ -328,17 +328,17 @@ legible in colorblind and grayscale modes.
 large scale factors for text size and display scaling.
 
 
-## More information
+## To learn more
```

Comment:
I think just "Learn more" might be more consistent with other headings on the site:

```suggestion
## Learn more
```